### PR TITLE
BaSP-TG: Admin delete and edit

### PR DESCRIPTION
**Pull request by LuisQueff**

### CODE REVIEW

File: src/resources/admins.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+const express = require('express');
+const fs = require('fs');
+const admins = require('../data/admins.json');
+
+const router = express.Router();
+
+router.delete('/delete/:id', (req, res) => {
+  const adminId = req.params.id;
+  const filteredadmins = admins.filter((admin) => admin.id !== adminId);
```
Comment (Juanoauil) [inline src/resources/admins.js:9]: write in camelCase

File: src/index.js [hunk 2]
```diff
@@ -6,8 +6,10 @@ const admins = require('./data/admins.json');
 
 const app = express();
 const port = process.env.PORT || 3000;
+const adminsRouter = require('./resources/admins');
```
Comment (Juanoauil) [inline src/index.js:9]: please move to line 6

File: src/resources/admins.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  fs.writeFile('src/data/admins.json', JSON.stringify(filteredadmins), (err) => {
+    if (err) {
+      res.send('Cannot delete admin');
+    } else {
+      res.send('admin deleted');
+    }
+  });
+});
+
+router.put('/change/:id', (req, res) => {
```
Comment (Juanoauil) [inline src/resources/admins.js:19]: replace path by /edit/:id

File: src/resources/admins.js (hunk 1 above)
Comment (LuisQueff) [reply src/resources/admins.js]: pronto

File: src/index.js (hunk 2 above)
Comment (LuisQueff) [reply src/index.js]: listo

File: src/resources/admins.js (hunk 3 above)
Comment (LuisQueff) [reply src/resources/admins.js]: ya esta

**Review by Juanoauil — DISMISSED** [review]

File: src/resources/admins.js [hunk 4]
```diff
@@ -4,8 +4,40 @@ const admins = require('../data/admins.json');
...
+      res.send('Cannot delete admin');
+    } else {
+      res.send('admin deleted');
+    }
+  });
+});
+
+router.put('/:id', (req, res) => {
+  const adminId = req.params.id;
+  const newadmin = req.body;
```
Comment (Juanoauil) [inline src/resources/admins.js:21]: please use camelCase

**Review by Juanoauil — APPROVED** [review]

**Pull request merged**